subsys/partition_manager/pm.yml.mcuboot

```diff
@@ -0,0 +1,10 @@
+#include <autoconf.h>
+
+mcuboot_sram:
+  size: CONFIG_PM_SRAM_SIZE
+  address: CONFIG_PM_SRAM_ADDRESS
```

Comment:
Is it valid to specify both a `span` and `size` + `address`? IIRC a partition which specifies a `span` is only a "logical" partition, in that it does not specify the amount of memory to reserve, it only gets its size and address derived from the partitions it spans.

Reply:
it's valid but its ignored, so I should probably remove it to avoid confusion.

Reply:
Then we should remove them as you say.